src/deep_learning_container.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Could you ping me out where this script been used?

Reply:
This will be used in place of this in next PR when the code will be merged (https://github.com/aws/deep-learning-containers/blob/master/tensorflow/training/docker/2.3.0/py3/Dockerfile.cpu#L167)

Reply:
I am Okay to merge current one first. However should we consider to squash them in one PR?

Reply:
I would prefer to merge this first so that we can use the url without any commit history on it.